apps/sms/services/js/moz_mobile_message/messaging_service.js

```diff
@@ -0,0 +1,191 @@
+/*global BridgeServiceMixin, Settings */
+'use strict';
+
+if (!('BridgeServiceMixin' in self)) {
+  importScripts('/services/js/bridge_service_mixin.js');
+}
+
+(function(exports) {
+
+const SERVICE_NAME = 'messaging-service';
+const MOBILE_MESSAGE_CLIENT_NAME = 'mozMobileMessageShim';
+
+const METHODS = Object.freeze([
+  'sendSMS', 'sendMMS', 'resendMessage'
+]);
+
+function ensureBridge() {
```

Comment:
I move it to here but I think we might this as utils for all the services in worker